src/index.html

```diff
@@ -1,13 +1,44 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images\imac.jpeg"
+        alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+        class="card__img card-img"
+      >
+
+      <p class="card__name name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+
+      <p class="card__code code">Product code: 195434</p>
+
+      <p class="card__reviews reviews">Reviews: 5</p>
```

Comment:
no needs to add new BEM block like name/code/reviews or something. You should have only two blocks card and stars